<!doctype html>
<html lang=en-us>

<head>
  <meta charset=utf-8>
  <meta name=viewport content="width=device-width,initial-scale=1">
  <meta http-equiv=X-UA-Compatible content="IE=edge">
  <meta name=generator content="Wowchemy 5.6.0 for Hugo">
  <link href=../static/css/styles.css rel=stylesheet>
  <meta name=description content>
  <link rel=alternate hreflang=en-us href=https://valadohaeris25.github.io/documentation/notebook />
  <meta name=theme-color content="#000000">
  <link rel=stylesheet href=../static/css/font-icon.css media=print onload='this.media="all"'>
  <link rel=stylesheet href=../static/css/academicons.min.css
    integrity="sha512-W0xM4mr6dEP9nREo7Z9z+9X70wytKvMGeDsj7ps2+xg5QPrEBXC8tAW1IFnzjR6eoJ90JmCnFzerQJTLzIEHjA=="
    crossorigin=anonymous media=print onload='this.media="all"'>
  <link rel=stylesheet href=../static/css/styles.css>
  <link rel=stylesheet href=../static/css/github-light.min.css title=hl-light media=print onload='this.media="all"'>
  <link rel=stylesheet href=../static/css/dracula.min.css title=hl-dark media=print onload='this.media="all"' disabled>
  <link rel=alternate href=/documentation/notebook/index.xml type=application/rss+xml title=SUMTHIMMUNOL_NMU>
  <link rel=icon type=image/png href=..//media/icon_hu2995325950933355676.png>
  <link rel=apple-touch-icon type=image/png href=/media/icon_hu2331250216965946225.png>
  <link rel=canonical href=https://valadohaeris25.github.io/documentation/notebook />
  <meta property="twitter:card" content="summary">
  <meta property="og:site_name" content="SUMTHIMMUNOL_NMU">
  <meta property="og:url" content="https://valadohaeris25.github.io/documentation/notebook/">
  <meta property="og:title" content="Notebook | SUMTHIMMUNOL_NMU">
  <meta property="og:description" content>
  <meta property="og:image" content="https://valadohaeris25.github.io/media/logo_hu16718177099216391851.png">
  <meta property="twitter:image" content="https://valadohaeris25.github.io/media/logo_hu16718177099216391851.png">
  <meta property="og:locale" content="en-us">
  <title>Notebook | SUMTHIMMUNOL_NMU</title>
  <link href=../static/css/bootstrap-icons.min.css rel=stylesheet>
</head>

<body id=top data-spy=scroll data-offset=70 data-target=#TableOfContents class="page-wrapper scroll_at_top"
  data-wc-page-id=4ce9d18b1664209cf4123f53b139b643>
  <script src=../static/js/wowchemy-init.min.1ee5462d74c6c0de1f8881b384ecc58d.js></script>
  <aside class=search-modal id=search>
    <div class=container>
      <section class=search-header>
        <div class="row no-gutters justify-content-between mb-3">
          <div class=col-6>
            <h1>Search</h1>
          </div>
          <div class="col-6 col-search-close"><a class=js-search href=# aria-label=Close><i
                class="fas fa-times-circle text-muted" aria-hidden=true></i></a></div>
        </div>
        <div id=search-box><input name=q id=search-query placeholder=Search... autocapitalize=off autocomplete=off
            autocorrect=off spellcheck=false type=search class=form-control aria-label=Search...></div>
      </section>
      <section class=section-search-results>
        <div id=search-hits></div>
      </section>
    </div>
  </aside>
  <div class=page-header>
    <header class=header--fixed>
      <nav class="navbar navbar-expand-lg navbar-light compensate-for-scrollbar" id=navbar-main>
        <div class=container-xl>
          <div class="d-none d-lg-inline-flex"><a class=navbar-brand href= ./../index.html><img
              src=./../static/picture/logo_hu14042517398967657203.png alt=SUMTHIMMUNOL_NMU></a></div><button type=button
            class=navbar-toggler data-bs-toggle=collapse data-bs-target=#navbar-content aria-controls=navbar-content
            aria-expanded=false aria-label="Toggle navigation">
            <span><i class="fas fa-bars"></i></span></button>
          <div class="navbar-brand-mobile-wrapper d-inline-flex d-lg-none">
            <a class=navbar-brand href= ./../index.html>
            <img
              src=./../static/picture/logo_hu14042517398967657203.png alt=SUMTHIMMUNOL_NMU>
            </a></div>
          <div class="navbar-collapse main-menu-item collapse justify-content-end" id=navbar-content>
            <ul class="navbar-nav d-md-inline-flex">
              <li class=nav-item><a class=nav-link href=./../index.html data-target=index.html><span>Home</span></a></li>
              <li class="nav-item dropdown"><a href=# class="nav-link dropdown-toggle" role=button data-bs-toggle=dropdown
                  aria-expanded=false><span>Project</span><span class=caret></span></a>
                <ul class=dropdown-menu>
                  <li><a class=dropdown-item href=./../project/background.html><span>Background</span></a></li>
                  <li><a class=dropdown-item href=./../project/design.html data-target=project/design><span>Design</span></a>
                  </li>
                  <li><a class=dropdown-item href=./../project/result.html data-target=project/result><span>Result</span></a>
                  </li>
                  <li><a class=dropdown-item href=./../project/report.pdf target="_blank"
                      ><span>Report</span></a></li>
                  <li><a class=dropdown-item href=./../project/supplementary.pdf target="_blank"
                      ><span>Supplementary information</span></a></li>
                </ul>
              </li>
              <li class="nav-item dropdown"><a href=# class="nav-link dropdown-toggle" role=button data-bs-toggle=dropdown
                  aria-expanded=false><span>Team</span><span class=caret></span></a>
                <ul class=dropdown-menu>
                  <li><a class=dropdown-item href=./../team/member.html data-target=team/member><span>Member</span></a></li>
                  <li><a class=dropdown-item href=./../team/community_contribution.html
                      data-target=team/community_contribution><span>Community contribution</span></a></li>
                </ul>
              </li>
              <li class="nav-item dropdown"><a href=# class="nav-link dropdown-toggle" role=button data-bs-toggle=dropdown
                  aria-expanded=false><span>Documentation</span><span class=caret></span></a>
                <ul class=dropdown-menu>
                  <li><a class=dropdown-item href=./../documentation/notebook.html
                      data-target=documentation/notebook><span>Notebook</span></a></li>
                  <li><a class=dropdown-item href=./../documentation/protocol.html
                      data-target=documentation/protocol><span>Protocol</span></a></li>
                </ul>
              </li>
            </ul>
          </div>
          <ul class="nav-icons navbar-nav flex-row ml-auto d-flex pl-md-2"></ul>
        </div>
      </nav>
    </header>
  </div>
  <div class="main-content-wrap page_documentation_notebook">
    <div class=page-body>
      <div class=page-docs-notebook>
        <div class=top-image><img src=./../static/picture/notebook-top.png></div>
        <div class="container">
          <div class="vue-content notebook-wrap">

            <div id="app_publications" data-v-app="">
              <div class="row">
                <div class="col-12 col-md-4 months-and-parts">
                  <div class="months-sep-wrap">
                    <ul class="months-list">
                      <li class="month-item">
                        <div class="img_box"><img src="./../static/images/note1.png" class="month-image"></div>
                      </li>
                      <li class="month-item">
                        <div class="img_box"><img src="./../static/images/note2.png" class="month-image"></div>
                      </li>
                      <li class="month-item">
                        <div class="img_box"><img src="./../static/images/note3.png" class="month-image"></div>
                      </li>
                      <li class="month-item">
                        <div class="img_box"><img src="./../static/images/note4.png" class="month-image"></div>
                      </li>
                      <li class="month-item">
                        <div class="img_box"><img src="./../static/images/note5.png" class="month-image"></div>
                      </li>
                    </ul>
                    <div class="sep2"><img src="../static/picture/sep2.png"></div>
                  </div>
                  <ul class="month-parts g-label-0">
                    <li class="part-item"><a class="month-part-item" href="#"></a></li>
                    <li class="part-item"><a class="month-part-item" href="#"></a></li>
                    <li class="part-item"><a class="month-part-item" href="#"></a></li>
                    <li class="part-item"><a class="month-part-item" href="#"></a></li>
                    <li class="part-item"><a class="month-part-item" href="#"></a></li>
                  </ul>
                </div>
                <div class="col-12 col-md-8">
                  <!-- <div class="note-content-wrap empty-init">
                    <div class="note-content empty-init-content">CLICK ON THE GREEN TAB TO VIEW THE INDIVIDUAL LAB RECORDS</div>
                  </div> -->
                  <div class="note-content-wrap">
                    <!-- 实验1: Determinate Binding Sites -->
                    <div class="note-content">
                      <p><strong>1. Experiment</strong>: Determinate Binding Sites</p>
                      <p><strong>2. Time</strong>: 2024.08.03-2024.08.07</p>
                      <p><strong>3. Member</strong>: Xudong Tang, Yang Jin, Binxuan Zhang, Kaiqing Zhang, Xuantong Liu
                      </p>
                      <p><strong>4. Method</strong>: Through reference research, we identified the native binding sites
                        for IL-2 receptor subunits β and γ as the active sites.</p>
                      <p><strong>5. Attachment</strong>: <a
                          href="..../uploads/docs/notebook/pdf/1-Determinate-Binding-Sites.pdf">Determinate Binding
                          Sites</a></p>
                    </div>

                    <!-- 实验2: RFDiffusion-Based Protein Design -->
                    <div class="note-content">
                      <p><strong>1. Experiment</strong>: RFDiffusion-Based Protein Design</p>
                      <p><strong>2. Time</strong>: 2024.08.29-2024.09.28</p>
                      <p><strong>3. Member</strong>: Xudong Tang, Yang Jin, Binxuan, Zhang, Kaiqing Zhang, Xuantong Liu
                      </p>
                      <p><strong>4. Method</strong>:</p>
                      <p>RFDiffusion is a cutting-edge method for de novo protein structure and function design,
                        leveraging the capabilities of deep learning and diffusion models. It is based on the
                        RoseTTAFold (RF) structure prediction network, fine-tuned on protein structure denoising tasks.
                        The principle behind RFDiffusion is to generate protein backbones by iteratively refining
                        noise-corrupted structures, eventually producing functional and structurally accurate proteins
                        from simple molecular specifications.</p>
                      <p>(1) Core Components:</p>
                      <p>① Denoising Diffusion Probabilistic Models (DDPMs): These models are trained to reverse the
                        process of adding Gaussian noise to protein structures, thereby generating new, realistic
                        protein structures.<br>
                        ② Rotational Equivariance: RFDiffusion maintains rotational equivariance, allowing it to model
                        three-dimensional (3D) structures in a global representation frame independent manner.<br>
                        ③ Conditioning Information: The model can be guided towards specific design objectives by
                        providing conditioning information at each step of the generation process.<br></p>
                      <p>(2) The procedure of RFDiffusion:</p>
                      <p>① Model Training</p>
                      <ul>
                        <li>Data Preparation: Sample protein structures from the Protein Data Bank (PDB) and introduce
                          noise to create training inputs.</li>
                        <li>Noise Application: Perturb Cα coordinates with 3D Gaussian noise and apply Brownian motion
                          to residue orientations.</li>
                        <li>Model Training: Train the RFDiffusion model by minimizing the mean-squared error (MSE) loss
                          between frame predictions and the true protein structure.</li>
                      </ul>
                      <p>② Protein Design</p>
                      <ul>
                        <li>Initialization: Start with random residue frames.</li>
                        <li>Denoising Iterations: Iteratively refine the protein structure by denoising the noisy input,
                          adding noise at each step to generate the input for the next iteration.</li>
                        <li>Sequence Design: Use the ProteinMPNN network to design sequences encoding the generated
                          protein structures.</li>
                      </ul>
                      <p>③ Conditioning for Specific Designs</p>
                      <ul>
                        <li>Unconditional Design: Generate diverse protein structures without additional input.</li>
                        <li>Topology-Constrained Design: Provide secondary structure and/or fold information to guide
                          the design towards specific topologies.</li>
                        <li>Symmetric Oligomer Design: Specify point group symmetry to create symmetric oligomeric
                          structures.</li>
                      </ul>
                      <p>④ Experimental Characterization</p>
                      <ul>
                        <li>Expression and Purification: Express the designed proteins in a suitable host and purify
                          them for further analysis.</li>
                        <li>Structural Verification: Use techniques such as circular dichroism (CD) and cryo-electron
                          microscopy (cryo-EM) to verify the structure and stability of the designed proteins.</li>
                        <li>Functional Validation: Assess the functionality of the designed proteins through binding
                          assays, enzymatic activity tests, or other relevant functional assays.</li>
                      </ul>
                      <p><strong>5. Attachment</strong>: <a
                          href="../uploads/docs/notebook/pdf/2-RFDiffusion-Based-Protein-Design.pdf">RFDiffusion Based
                          Protein Design</a></p>
                    </div>

                    <!-- 实验3: Using HDOCK to evaluating the binding energy -->
                    <div class="note-content">
                      <p><strong>1. Experiment</strong>: Using HDOCK to evaluating the binding energy</p>
                      <p><strong>2. Time</strong>: 2024.09.10-2024.09.12</p>
                      <p><strong>3. Member</strong>: Xudong Tang, Yang Jin, Binxuan Zhang, Kaiqing Zhang, Xuantong Liu
                      </p>
                      <p><strong>4. Method</strong>:</p>
                      <p>HDock is an efficient protein-protein interaction docking tool that combines template matching
                        and free docking algorithms to accurately predict intermolecular interactions. During the HDock
                        precision screening phase, binding score calculations consider structural alignment between bait
                        proteins and target proteins, enabling more effective identification of protein interactions
                        with enhanced stability and higher binding affinity. This serves as a crucial screening metric
                        that improves result accuracy.</p>
                      <p>(1) The procedure of HDock:</p>
                      <p>① Data input: The HDOCK web server is available at <a href="http://hdock.phys.hust.edu.cn/"
                          target="_blank" rel="noopener">http://hdock.phys.hust.edu.cn/</a>. Both sequence and structure
                        information of ligand and receptor should be uploaded through the website separately. The
                        docking process will be carried out in the server. Click submit to wait for the results.
                        Generally, the waiting time is approximately 30 minutes. The submitted sequences should be in
                        FASTA format and contain only standard amino acids. The structure should be in PDB format. In
                        addition, residue information of the binding site is also possible to be submitted as a filter
                        in the docking process.</p>
                      <p>② Docking preparation: First, the homologous sequences of molecules will be searched by
                        sequence similarity search. The template with the highest sequence coverage, highest sequence
                        similarity and highest resolution will be selected to build the template-based model. ClustalW
                        is used for sequence alignment, and MODELLER is used for model construction. This step will be
                        executed on the server.</p>
                      <p>③ Global docking: HDOCKlite, a layered docking program based on FFT, is used to globally sample
                        the presumed binding directions. We subjected the screened proteins to HDock refined screening,
                        where adjusting the Spacing and Angle parameters to 1.2 and 15 provides highly accurate docking
                        results, offering data references for protein-protein interaction strength. Docking scores are
                        generally negative values, with more negative scores indicating a higher probability of valid
                        binding models. The ranked binding modes are clustered with an RMSD cutoff of 5 Å. if two
                        binding modes have a ligand RMSD of ≤ 5 Å, the one with the better score is kept. This step will
                        be executed on the server. The docking model of HDOCKlite and the template-based docking model
                        of MODELLER will be provided for download through web interaction.</p>
                      <p>④ Results analysis: The server will send the top 10 docking models to the website. The results
                        page contains an interactive visualization window and a summary table. The summary table usually
                        contains docking energy score, confidence score and ligand RMSD. If the SAXS data file is
                        provided, the SAXS chi-square of the model can also be displayed. Results can also be downloaded
                        as a PDB file, using the molecular visualization software PyMOL to open and view the predicted
                        model. The docking results of HDock are generally referenced by the docking score and input into
                        the formula: Confidence_score = 1.0/[1.0 + e^(0.02*(Docking_Score+150))]. Taking a docking score
                        of -200 as a reference, the confidence score at this point is approximately 0.7. In the official
                        documentation, a confidence score > 0.7 (closer to 1) indicates a high likelihood that the two
                        protein molecules will bind; a confidence score between 0.5 and 0.7 indicates that the two
                        proteins may bind; a confidence score < 0.5 indicates that it is unlikely that the two molecules
                          will bind. Therefore, a docking score lower than -200 corresponds to a very high probability
                          of binding.</p>
                          <p><strong>5. Attachment</strong>: <a
                              href="../uploads/docs/notebook/pdf/3-Using-HDOCK-to-evaluating-the-binding-energy.pdf">Using
                              HDOCK to evaluating the binding energy</a></p>
                    </div>

                    <!-- 实验4: Hydrogen bond depiction -->
                    <div class="note-content">
                      <p><strong>1. Experiment</strong>: Hydrogen bond depiction</p>
                      <p><strong>2. Time</strong>: 2024.09.30-2024.10.05</p>
                      <p><strong>3. Member</strong>: Fan Yang, Qiwen Jiang, Men Sun, Xinxin Zhang, Ziyan Yu</p>
                      <p><strong>4. Method</strong>:</p>
                      <p>(1) The PDB file for the protein is prepared.</p>
                      <p>(2) In PyMol, chains should be renamed. Then, hydrogen bonds are found by clicking
                        "find-polar-contacts-to other atoms in object". The stick structures of the protein molecules
                        are displayed. After the amino acids connected by hydrogen bonds are clicked, we show the
                        cartoon figure of the molecules and show the stick structure of the amino acids clicked. Then,
                        the sequence numbers and names of the amino acids can be annotated by clicking on
                        "label-residue".</p>
                      <p>(3) Click "draw/ray-save image to file".</p>
                      <p><strong>5. Attachment</strong>: <a
                          href="../uploads/docs/notebook/pdf/4-Hydrogen-bond-depiction.pdf">Hydrogen bond depiction</a>
                      </p>
                    </div>

                    <!-- 实验5: The optimization of sequence -->
                    <div class="note-content">
                      <p><strong>1. Experiment</strong>: The optimization of sequence</p>
                      <p><strong>2. Time</strong>: 2024.10.04-2024.10.05</p>
                      <p><strong>3. Member</strong>: Xudong Tang, Qiwen Jiang, Binxuan Zhang, Xuantong Liu</p>
                      <p><strong>4. Method</strong>:</p>
                      <p>(1) Notably, although the candidate sequences, specifically B0, B34, and B51, along with G9,
                        G16, G35, G42, G57, and G71, engaged the binding interface via fewer than four amino acids, a
                        substantial portion of their sequences remained uninvolved in IL-2 binding. Consequently, these
                        sequences were truncated to retain only the 20 amino acids implicated in binding interface
                        contacts, with the exception of B34, which was truncated to 39 amino acids for further
                        investigation (Table.1, 2).</p>
                      <p>(2) In PyMOL, based on the pre-existing file, we clicked "Display" → "Sequence", selected the
                        corresponding amino acid residues, and executed "Remove Atoms" to generate the result shown in
                        the figure below.</p>
                      <p><strong>5. Attachment</strong>: <a
                          href="../uploads/docs/notebook/pdf/5-The-optimization-of-sequence.pdf">The optimization of
                          sequence</a></p>
                    </div>

                    <!-- 实验6: Sequence design -->
                    <div class="note-content">
                      <p><strong>1. Experiment</strong>: Sequence design</p>
                      <p><strong>2. Time</strong>: 2024.10.05-2024.10.13</p>
                      <p><strong>3. Member</strong>: Fan Yang, Qiwen Jiang, Xinxin Zhang, Kaiqing Zhang, Meng Sun</p>
                      <p><strong>4. Method</strong>:</p>
                      <p>(1) In this study, a homologous recombination strategy was employed for the experimental
                        design. The selected peptide sequences, namelyB0, B34, B51 and G9, G16, G35, G42, G57, G71,
                        along with their corresponding reverse sequences, were conjugated via a flexible linker. Reverse
                        sequences were incorporated due to the flexibility of peptide linker and the inherent
                        directionality of protein-receptor binding. Furthermore, glycosylphosphatidylinositol a (GPI)
                        membrane-anchoring sequence was fused to the construct to retain the IL-2 mimetics on the cell
                        surface rather than their secretion into the extracellular space, facilitating activation of NK
                        cells. Additionally, Flag tag was coupled with the recombinant sequences for subsequent
                        detection and isolation purposes.</p>
                      <p>(2) The specific sequences of the genetic elements are listed in the table below.</p>
                      <p><strong>5. Attachment</strong>: <a
                          href="../uploads/docs/notebook/pdf/6-Sequence-design.pdf">Sequence design</a></p>
                    </div>

                    <!-- 实验7: Lentivirus Packaging and Infection of CAR on NK-92 cell line -->
                    <div class="note-content">
                      <p><strong>1. Experiment</strong>: Lentivirus Packaging and Infection of CAR on NK-92 cell line
                      </p>
                      <p><strong>2. Time</strong>: 2024.10.26-2024.12.01</p>
                      <p><strong>3. Member</strong>: Fan Yang, Qiwen Jiang, Xinxin Zhang, Kaiqing Zhang, Meng Sun</p>
                      <p><strong>4. Principle</strong>: Lentiviral packaging is a technique to generate
                        replication-incompetent recombinant lentiviral particles for efficient delivery of exogenous
                        genes into target cells. Its core principle relies on the split of viral functional components
                        into separate plasmids, which are co-expressed in packaging cells to assemble infectious but
                        non-replicating viral particles.</p>
                      <p><strong>5. Materials</strong>: Lenti-293 T cells, 5% CO2 incubator, 5% FBS, CPT Transfection
                        Kit (Viraltherapy), Centrifuge 5810 R (Eppendorf, 5810R), Eclipse TS100 (Nikon, TS100), 96-well
                        plates, 5% CO2 incubator, complete medium (RPMI-1640 with 10% FBS), polybrene.</p>
                      <p><strong>6. Method</strong>:</p>
                      <p>(1) One day before transfection, Lenti-293 T cells was passaged and evenly plated on 100 mm
                        culture dishes (at 8-10×106 cells per dish). Cells were cultured at 37 °C in a 5% CO2 incubator.
                        The medium was replaced with 10.5 mL of pre-warmed, resistance-free medium (5% FBS) 2-4 hours
                        before transfection.</p>
                      <p>(2) A mixture of twenty-three plasmids in equimolar amounts was prepared, and transfection was
                        performed using the calcium phosphate transfection method with the CPT Transfection Kit
                        (Viraltherapy). The transfection mixture composition was as follows:</p>
                      <p><strong>7. Attachment</strong>: <a
                          href="../uploads/docs/notebook/pdf/7-Lentivirus-Packaging-and-Infection-of-CAR-on-NK-92-cell-line.pdf">Lentivirus
                          Packaging and Infection of CAR on NK 92 cell line</a></p>
                    </div>

                    <!-- 实验8: Single-Cell Sorting and Planting -->
                    <div class="note-content">
                      <p><strong>1. Experiment</strong>: Single-Cell Sorting and Planting</p>
                      <p><strong>2. Time</strong>: 2024.12.21-2024.12.28</p>
                      <p><strong>3. Member</strong>: Fan Yang, Qiwen Jiang, Xinxin Zhang, Kaiqing Zhang, Meng Sun</p>
                      <p><strong>4. Principle</strong>: Fluorescence-activated cell sorting (FACS) uses target cells'
                        specific fluorescent signals to isolate individual cells into 96-well plates, ensuring clonal
                        growth from a single cell. Second, in vitro culture mimics physiological conditions to sustain
                        cell viability and proliferation, while inverted microscopy tracks clonal expansion. For
                        sequencing, gentle centrifugation preserves cell integrity during collection; cell lysis
                        releases nucleic acids, enabling high-quality sequencing of clonally pure single-cell samples.
                      </p>
                      <p><strong>5. Materials</strong>: NK cell media (Gibco, supplemented with 5% FBS and 500 IU/mL
                        IL-2), IL-2 (Gibco), 5% hAB serum (Gibco), Trypan Blue</p>
                      <p><strong>6. Attachment</strong>: <a
                          href="../uploads/docs/notebook/pdf/8-Single-Cell-Sorting-and-Planting.pdf">Single-Cell Sorting
                          and Planting</a></p>
                    </div>

                    <!-- 实验9: BLI measurement of cytokine-receptor binding affinity -->
                    <div class="note-content">
                      <p><strong>1. Experiment</strong>: BLI measurement of cytokine-receptor binding affinity</p>
                      <p><strong>2. Time</strong>: 2025.05.26-2025.05.31</p>
                      <p><strong>3. Member</strong>: Xudong Tang, Yang Jin, Binxuan Zhang, Kaiqing Zhang, Xuantong Liu
                      </p>
                      <p><strong>4. Principle</strong>: Bio-Layer Interferometry (BLI) is a label-free detection
                        technology that converts optical interference signals generated on the surface of BLI biosensors
                        into real-time response signals for detection. Its fundamental principle is wave interference.
                        The biosensor consists of a glass optical fiber layer and an optical layer. At the interface
                        between the optical fiber layer and the optical layer, the first reflected light is generated,
                        while the second reflected light is produced at the interface between the optical layer and the
                        external solution. The superposition of these two waves forms an interference spectrum curve.
                        When the bait binds to the ligand immobilized on the sensor surface, the thickness of the
                        optical layer increases, causing the path length of the second reflected light to become longer
                        than before. This results in a change in the interference spectrum curve, shifting it to the
                        right. When the bait and ligand dissociate, the bait detaches from the biosensor surface into
                        the solution, causing the interference spectrum curve to shift to the left. A plot of the shift
                        distance of the interference spectrum curve against the reaction time is called a sensorgram.
                        Based on various binding models, the association constant (Ka or Kon), dissociation constant (Kd
                        or Koff), and affinity (KD) can be fitted. Using the sensorgram, fitted curves, and affinity
                        kinetic parameters, the affinity of designed IL-2R agonists for IL-2Rβ and IL-2Rγ can be
                        evaluated and ranked.</p>
                      <p>Biosensors offer diverse methods for immobilizing bait molecules, including amine coupling,
                        biotin/streptavidin, antibody/anti-Fc, His-tag/Ni-NTA, GST/anti-GST, or FLAG/anti-FLAG
                        interactions, to form a biomolecular layer on the tip of the fiber optic biosensor. In this
                        experiment, due to the widespread use, high affinity, and effectiveness of SA sensors, we
                        selected SA sensors to immobilize the bait IL-2Rβ or IL-2Rγ for analyzing the affinity of IL-2R
                        agonists. SA sensors utilize the interaction between streptavidin and biotin. Therefore, we
                        purchased IL-2Rβ and IL-2Rγ with a His-tag (for purification) from Novoprotein, with the
                        sequence number C33A. In subsequent operations, these were used as bait and pre-biotinylated</p>
                      <p><strong>5. Materials</strong>: SA Sensors, Greiner 96-well Black Plate, PBST Buffer (0.02%
                        Tween 20), 1.5 mL EP Tubes, 15 mL Centrifuge Tube, Empty Sensor Tray, Octet® RED96 BLI
                        instrument, IL-2Rβ, IL-2Rγ, IL-2 mimics</p>
                      <p><strong>6. Attachment</strong>: <a
                          href="../uploads/docs/notebook/pdf/9-BLI-measurement-of-cytokine-receptor-binding-affinity.pdf">BLI
                          measurement of cytokine-receptor binding affinity</a></p>
                    </div>

                    <!-- 实验10: Molecular dynamics simulation -->
                    <div class="note-content">
                      <p><strong>1. Experiment</strong>: Molecular dynamics simulation</p>
                      <p><strong>2. Time</strong>: 2025.05.10-2025.05.29</p>
                      <p><strong>3. Member</strong>: Xudong Tang, Yang Jin, Kaiqing Zhang, Binxuan Zhang, Xuantong Liu
                      </p>
                      <p><strong>4. Materials</strong>:</p>
                      <p>(1) Software: GROMACS 2022.5, Alphafold3, HADDOCK, Visual Molecular Dynamics (VMD)</p>
                      <p>(2) Database: Protein Data Bank (PDB)</p>
                      <p>(3) Force Fields: CHARMM36m, TIP3P, CHARMM</p>
                      <p>(4) Simulation Environment: Periodic Cubic Water Box, NaCl</p>
                      <p><strong>5. Attachment</strong>: <a
                          href="../uploads/docs/notebook/pdf/10-Molecular-dynamics-simulation.pdf">Molecular dynamics
                          simulation</a></p>
                    </div>

                    <!-- 实验11: Cell proliferation assessment by cell counting method -->
                    <div class="note-content">
                      <p><strong>1. Experiment</strong>: Cell proliferation assessment by cell counting method</p>
                      <p><strong>2. Time</strong>: 2025.02.10-2025.02.29</p>
                      <p><strong>3. Member</strong>: Xudong Tang, Yang Jin, Binxuan Zhang, Kaiqing Zhang, Xuantong Liu
                      </p>
                      <p><strong>4. Materials</strong>: Cell culture flasks, automated cell counter, pipettes and
                        pipette tips, cell culture medium, incubator, centrifuge, cell culture dishes, sterile tubes or
                        containers, growth curve plotting software, IL-2</p>
                      <p><strong>5. Attachment</strong>: <a
                          href="../uploads/docs/notebook/pdf/11-Cell-proliferation-assessment-by-cell-counting-method-under-acidic-condition.pdf">Cell
                          proliferation assessment by cell counting method under acidic condition</a></p>
                    </div>

                    <!-- 实验12: LDH assay for measuring cytotoxic activity -->
                    <div class="note-content">
                      <p><strong>1. Experiment</strong>: LDH assay for measuring cytotoxic activity</p>
                      <p><strong>2. Time</strong>: 2024.05.26-2024.05.31</p>
                      <p><strong>3. Member</strong>: Xudong Tang, Yang Jin, Binxuan Zhang, Kaiqing Zhang, Xuantong Liu
                      </p>
                      <p><strong>4. Materials</strong>: K562 cell line, Microplate reader, CO2 incubator, LDH assay kit,
                        96-well plate, Pipette</p>
                      <p><strong>5. Attachment</strong>: <a
                          href="../uploads/docs/notebook/pdf/12-LDH-assay-for-measuring-cytotoxic-activity.pdf">LDH assay
                          for measuring cytotoxic activity</a></p>
                    </div>

                    <!-- 实验13: Flow Cytometric Analysis of NK Cell Activity -->
                    <div class="note-content">
                      <p><strong>1. Experiment</strong>: Flow Cytometric Analysis of NK Cell Activity</p>
                      <p><strong>2. Time</strong>: 2025.05.10-2025.05.29</p>
                      <p><strong>3. Member</strong>: Xudong Tang, Yang Jin, Binxuan Zhang, Kaiqing Zhang, Xuantong Liu
                      </p>
                      <p><strong>4. Materials</strong>: Flow cytometer, CD107a-APC, CD56-PE, CD69-FITC, Ice-cold PBS
                        Staining buffer, Brefeldin A, 4% paraformaldehyde (PFA), 0.1% Triton X-100</p>
                      <p><strong>5. Attachment</strong>: <a
                          href="../uploads/docs/notebook/pdf/13-Flow-Cytometric-Analysis-of-NK-Cell-Activity.pdf">Flow
                          Cytometric Analysis of NK Cell Activity</a></p>
                    </div>

                    <!-- 实验14: Flow Cytometric Analysis of NK Cell PFN & GrB -->
                    <div class="note-content">
                      <p><strong>1. Experiment</strong>: Flow Cytometric Analysis of NK Cell PFN & GrB</p>
                      <p><strong>2. Time</strong>: 2025.05.10-2025.05.29</p>
                      <p><strong>3. Member</strong>: Xudong Tang, Yang Jin, Binxuan Zhang, Kaiqing Zhang, Xuantong Liu
                      </p>
                      <p><strong>4. Materials</strong>: Flow cytometer, Anti-human Perforin antibody (BioLegend,
                        308104), Anti-Human Granzyme B Antibody (BioLegend, 515404),Ice-cold PBS Staining buffer,
                        Brefeldin A, 4% paraformaldehyde (PFA), 0.1% Triton X-100</p>
                      <p><strong>5. Attachment</strong>: <a
                          href="../uploads/docs/notebook/pdf/14-Flow-Cytometric-Analysis-of-NK-Cell-PFN-&-GrB.pdf">Flow
                          Cytometric Analysis of NK Cell PFN & GrB</a></p>
                    </div>

                    <!-- 实验15: Software structure prediction and visual inspection -->
                    <div class="note-content">
                      <p><strong>1. Experiment</strong>: Software structure prediction and visual inspection</p>
                      <p><strong>2. Time</strong>: 2024.06.07-2024.06.15</p>
                      <p><strong>3. Member</strong>: Xudong Tang, Yang Jin, Binxuan Zhang, Xuantong Liu, Kaiqing Zhang
                      </p>
                      <p><strong>4. Method</strong>:</p>
                      <p>(1) Alphafold3 structure prediction</p>
                      <p>① Open the website, find an interface for inputting the protein sequence. Select "protein" for
                        all options, and then input data.</p>
                      <p>② Initially, input IL-2Rβ sequence and IL-2Rγ sequence. These two sequences are always
                        consistent and no need to be changed in subsequent steps.</p>
                      <p>③ For the third entry, input our predicted protein structure: β + linker + γ. Through
                        sequencing, we selected three complexes B34G35R-G, B51G35R-G, and B51G9-G, and chose GALA
                        peptide which repeat four to six times as linker to connect protein fragments. Through arranging
                        and combining them, we got 9 combinations.</p>
                      <p>④ Evaluate the predicted protein structures through scores and predicted 3D model.</p>
                      <p>(2) Visual inspection</p>
                      <p>① Notably, we use the candidate sequences, specifically B34, and 51, along with G9 and 35.
                        These sequences were truncated to retain the 20 amino acids implicated in binding interface
                        contacts, with the exception of β34, which was truncated to 39 amino acids for further
                        investigation.</p>
                      <p>② In PyMOL chains should be renamed. Then, hydrogen bonds are found by clicking
                        "find-polar-contacts-to other atoms in object". The stick structures of the protein molecules
                        are displayed. After the amino acids connected by hydrogen bonds are clicked, we show the
                        cartoon figure of the molecules and show the stick structure of the amino acids clicked. Then,
                        the sequence numbers and names of the amino acids can be annotated by clicking on
                        "label-residue".</p>
                      <p>③ Click " draw/ray-save image to file "</p>
                      <p><strong>5. Attachment</strong>: <a
                          href="../uploads/docs/notebook/pdf/15-Software-structure-prediction-and-visual-inspection.pdf">Software
                          structure prediction and visual inspection</a></p>
                    </div>

                    <!-- 实验16: BLI-based affinity assessment of IL-2 Mimics under acidic conditions -->
                    <div class="note-content">
                      <p><strong>1. Experiment</strong>: BLI-based affinity assessment of IL-2 Mimics under acidic
                        conditions</p>
                      <p><strong>2. Time</strong>: 2025.06.26-2025.07.10</p>
                      <p><strong>3. Member</strong>: Xudong Tang, Yang Jin, Binxuan Zhang, Kaiqing Zhang, Xuantong Liu
                      </p>
                      <p><strong>4. Materials</strong>: SA Sensors, Greiner 96-well Black Plate, 0.5M MES buffer
                        (pH=6.0, J62574.AK 2-Morpholinoethanesulfonic acid sodium salt: 71119-23-8), 1.5 mL EP Tubes, 15
                        mL Centrifuge Tube, Empty Sensor Tray, Octet RED96 BLI instrument, IL-2Rβγ, IL-2 mimics(GALA
                        peptide conjugation)</p>
                      <p><strong>5. Attachment</strong>: <a
                          href="../uploads/docs/notebook/pdf/16-BLI-based-affinity-assessment-of-IL-2-Mimics-under-acidic-conditions.pdf">BLI-based
                          affinity assessment of IL-2 Mimics under acidic conditions</a></p>
                    </div>

                    <!-- 实验17: Detect the activating signal pathway by Western Blotting under acidic condition -->
                    <div class="note-content">
                      <p><strong>1. Experiment</strong>: Detect the activating signal pathway by Western Blotting under
                        acidic condition</p>
                      <p><strong>2. Time</strong>: 2025.07.08-2025.07.14</p>
                      <p><strong>3. Member</strong>: Xudong Tang, Yang Jin, Binxuan Zhang, Kaiqing Zhang, Xuantong Liu
                      </p>
                      <p><strong>4. Materials</strong>: Lysis solution, PBS, protein loading buffer, electrophoresis
                        buffer, transfer buffer, blocking buffer, centrifugal tube, electrophoresis tank, transfer
                        membrane device, NC membrane, β-actin antibody (CST, 4970) , Phospho-Jak1 antibody (CST,74129),
                        Jak1 antibody (CST, 29261), Phospho-Jak3 antibody (CST, 5031), JAK3 antibody (CST,80331),
                        Phospho-ERK antibody (CST), ERK antibody (CST), Phospho-STAT1 (CST,9167), STAT1 antibody
                        (CST,14994), Phospho-STAT3 (CST, 9145), STAT3 (CST, 9139), Phospho-AKT (CST, 9271), T-AKT
                        (CST,4691)</p>
                      <p><strong>5. Attachment</strong>: <a
                          href="../uploads/docs/notebook/pdf/17-Detect-the-activating-signal-pathway-by-Western-Blotting-under-acidic-condition.pdf">Detect
                          the activating signal pathway by Western Blotting under acidic condition</a></p>
                    </div>

                    <!-- 实验18: Cell proliferation assessment by cell counting method under acidic condition -->
                    <div class="note-content">
                      <p><strong>1. Experiment</strong>: Cell proliferation assessment by cell counting method under
                        acidic condition</p>
                      <p><strong>2. Time</strong>: 2025.07.10-2025.07.29</p>
                      <p><strong>3. Member</strong>: Xudong Tang, Yang Jin, Binxuan Zhang, Kaiqing Zhang, Xuantong Liu
                      </p>
                      <p><strong>4. Materials</strong>: Cell culture flasks, automated cell counter, pipettes and
                        pipette tips, cell culture medium, incubator, centrifuge, cell culture dishes, sterile tubes or
                        containers, growth curve plotting software, IL-2 , lactic acid</p>
                      <p><strong>5. Attachment</strong>: <a
                          href="../uploads/docs/notebook/pdf/18-Cell-proliferation-assessment-by-cell-counting-method-under-acidic-condition.pdf">Cell
                          proliferation assessment by cell counting method under acidic condition</a></p>
                    </div>

                    <!-- 实验19: LDH assay for measuring cytotoxic activity under acidic condition -->
                    <div class="note-content">
                      <p><strong>1. Experiment</strong>: LDH assay for measuring cytotoxic activity under acidic
                        condition</p>
                      <p><strong>2. Time</strong>: 2024.08.26-2024.08.31</p>
                      <p><strong>3. Member</strong>: Xudong Tang, Yang Jin, Binxuan Zhang, Kaiqing Zhang, Xuantong Liu
                      </p>
                      <p><strong>4. Materials</strong>: K562 cell line, Microplate reader, CO2 incubator, LDH assay kit,
                        96-well plate, Pipette</p>
                      <p><strong>5. Attachment</strong>: <a
                          href="../uploads/docs/notebook/pdf/19-LDH-assay-for-measuring-cytotoxic-activity-under-acidic-condition.pdf">LDH
                          assay for measuring cytotoxic activity under acidic condition</a></p>
                    </div>

                    <!-- 实验20: Flow Cytometric Analysis of NK Cell Activity under acidic condition -->
                    <div class="note-content">
                      <p><strong>1. Experiment</strong>: Flow Cytometric Analysis of NK Cell Activity under acidic
                        condition</p>
                      <p><strong>2. Time</strong>: 2025.09.10-2025.09.20</p>
                      <p><strong>3. Member</strong>: Xudong Tang, Yang Jin, Binxuan Zhang, Kaiqing Zhang, Xuantong Liu
                      </p>
                      <p><strong>4. Principle</strong>: This experiment uses flow cytometry for multi-parameter cell
                        analysis to explore how acidic conditions affect NK cell activity. NK cells are identified via
                        CD56-PEas CD3⁻CD56⁺ populations. CD107serves as an activation marker: surface expression
                        indicates cytotoxicity (CD56⁺CD107a⁺ = active NK cells). Flow cytometry excludes doublets/dead
                        cells, then quantifies the ratio of active NK cells to total NK cells to assess functional
                        impacts.</p>
                      <p><strong>5. Materials</strong>: Flow cytometer, CD107a-APC, CD56-PE, Ice-cold PBS Staining
                        buffer, Brefeldin A, 4% paraformaldehyde (PFA), 0.1% Triton X-100, 15 mM lactic acid, K562 cell
                        line</p>
                      <p><strong>6. Attachment</strong>: <a
                          href="../uploads/docs/notebook/pdf/20-Flow-Cytometric-Analysis-of-NK-Cell-Activity-under-acidic-condition.pdf">Flow
                          Cytometric Analysis of NK Cell Activity under acidic condition</a></p>
                    </div>

                    <!-- 实验21: Flow cytometric analysis of NK cell PFN & GrB under acidic condition -->
                    <div class="note-content">
                      <p><strong>1. Experiment</strong>: Flow cytometric analysis of NK cell PFN & GrB under acidic
                        condition</p>
                      <p><strong>2. Time</strong>: 2025.09.10-2025.09.15</p>
                      <p><strong>3. Member</strong>: Xudong Tang, Yang Jin, Binxuan Zhang, Kaiqing Zhang, Xuantong Liu
                      </p>
                      <p><strong>4. Materials</strong>: Flow cytometer, Anti-human Perforin antibody (BioLegend 308104),
                        anti-Human Granzyme B Antibody (BioLegend 515404), Ice-cold PBS Staining buffer, Brefeldin A, 4%
                        paraformaldehyde (PFA), 0.1% Triton X-100</p>
                      <p><strong>5. Attachment</strong>: <a
                          href="../uploads/docs/notebook/pdf/21-Flow-cytometric-analysis-of-NK-cell-PFN-&-GrB-under-acidic-condition.pdf">Flow
                          cytometric analysis of NK cell PFN & GrB under acidic condition</a></p>
                    </div>
                  </div>
                </div>
              </div>
            </div>

          </div>
        </div>
        <!--  -->
      </div>
    </div>
    <div class=page-footer>
      <div class=container>
        <div class=nav-footer>
          <div class=nav-footer-center>&nbsp;</div>
          <div class=nav-footer-right>&nbsp;</div>
        </div>
      </div>
    </div>
    <div id=back_top_img_btn class=back_top><img src=./../static/picture/goup.png></div>
  </div>
  <script src=../static/js/vendor-bundle.min.c87a3ee09b90a2d201ed503217de637d.js></script>
  <script id=search-hit-fuse-template type=text/x-template>
    <div class="search-hit" id="summary-{{key}}">
      <div class="search-hit-content">
        <div class="search-hit-name">
          <a href="{{relpermalink}}">{{title}}</a>
          <div class="article-metadata search-hit-type">{{type}}</div>
          <p class="search-hit-description">{{snippet}}</p>
        </div>
      </div>
    </div>
  </script>
  <script src=../static/js/fuse.min.js
    integrity="sha512-o38bmzBGX+hD3JHWUFCDA09btWaqrNmoJ3RXLlrysA7PP01Kgs4UlE4MhelE1v5dJR3+cxlR4qQlotsW7jKsnw=="
    crossorigin=anonymous></script>
  <script src=../static/js/jquery.mark.min.js
    integrity="sha512-mhbv5DqBMgrWL+32MmsDOt/OAvqr/cHimk6B8y/bx/xS88MVkYGPiVv2ixKVrkywF2qHplNRUvFsAHUdxZ3Krg=="
    crossorigin=anonymous></script>
  <script id=page-data type=application/json>{"use_headroom":true}</script>
  <script src=../static/js/wowchemy-headroom.c251366b4128fd5e6b046d4c97a62a51.js type=module></script>
  <script src=../static/js/wowchemy.min.da4dbf53e12727fb8e8a8b2f9a8483f4.js></script>
  <script src="../static/js/jquery.min.js"></script>
  <script src="../static/js/script.js"></script>
</body>

</html>
